src/Analyzers/CSharp/Tests/RemoveUnnecessaryCast/RemoveUnnecessaryCastTests.cs

```diff
@@ -142,17 +142,16 @@ static void Main()
         }
 
         [WorkItem(545138, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/545138")]
```

Comment:
@sharwell @CyrusNajmabadi Would you be able to check the original work item if it exists?
I think the original test unintentionally had these compile errors. Then the test was probably moved to the new testing library and the expected diagnostics were added just to satisfy the test and make it pass.

Reply:
The issue does not contain the compile errors.

Reply:
@sharwell Thanks for confirmation. This should probably have happened accidentally.